docs/docs/changelog.md

```diff
@@ -5,6 +5,11 @@ sidebar_position: 7
 
 # Changelog
 
+## Unreleased
+
+- Added new `splitArgs` to the template system
+  (`{{splitArgs "foo bar 'foo bar baz'"}}`) to ensure string is splitted as sh arguments not whitespaces
+
```

Comment:
It's not necessary to update this file in your PR. This file will be updated from the `CHANGELOG.md` when we release a new version.